db/organizations/pingdom.eno

```diff
@@ -1,12 +1,12 @@
-name: Pingdom
-website_url: https://www.pingdom.com/
-privacy_policy_url: https://www.pingdom.com/legal/privacy-policy/
-privacy_contact: support@pingdom.com
-country: 
-description: "Real User Monitoring let's you know exactly how real visitors experience your website. Analyze performance data and how it relate to browsers, platforms and even countries, all in real-time. Improve your customers experience now."
-| 
+name: SolarWinds
```

Comment:
Same comment as for loggly, which has also been acquired by SolarWinds. For now, please keep the entry, except for the notes.

I split it out (same issue as linked above): https://github.com/ghostery/trackerdb/issues/176